src/service/config.rs

```diff
@@ -33,6 +36,39 @@ impl Default for ServiceConfig {
     }
 }
 
+#[derive(Clone, Copy)]
```

Comment:
This looks like a refactoring, it's better in its own PR.

Reply:
This involves the external interface of `yamux`, which must be merged before the release.

Reply:
https://github.com/nervosnetwork/p2p/pull/209